pkg/cli/util.go

```diff
@@ -0,0 +1,78 @@
+package cli
+
+import (
+	"net"
+
+	"github.com/pkg/errors"
+	"github.com/replicatedhq/kurl/pkg/netutils"
+	"github.com/spf13/cobra"
+)
+
+func newUtilCommand(cli CLI) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "util",
+		Short: "Utility commands",
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+			return cli.GetViper().BindPFlags(cmd.PersistentFlags())
+		},
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return cli.GetViper().BindPFlags(cmd.Flags())
+		},
+	}
+	return cmd
+}
+
+func newUtilIfaceFromIPCommand(cli CLI) *cobra.Command {
```

Comment:
```suggestion
func newUtilIfaceFromIPCommand(_ CLI) *cobra.Command {
```

To fix the lint issue